common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @buccaneer
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = buccaneer
+
+	flag = global_maa
+}
+
+innovation_brute = {
```

Comment:
I would give gnolls hills bonus too since they dwell there too.

Reply:
I see the hills point like for Kobold, but still there are a lot of them in Elwyn no?

Reply:
> I see the hills point like for Kobold and I will add, but still, there are a lot of them in Elwyn no?

Kobolds? Talking about gnolls here. Gnolls mostly live in forests, yeah.

Reply:
I was saying, I understand this point, just like you Kobold comment, regardless, what is the verdict?

Reply:
> I was saying, I understand this point, just like your Kobold comment, regardless, what is the verdict?

Add hills bonus? I said it in the first comment.